improver_tests/acceptance/test_regrid.py

```diff
@@ -45,7 +45,7 @@
 
 def test_regrid_basic(tmp_path):
     """Test basic regridding"""
-    # KGO for this test is the same as test_regrid_check_landmask below
+    # KGO for this test (deafult: bilinear) is the same as test_regrid_bilinear_2
```

Comment:
Suggest minor spelling/typo fix:
```suggestion
    # KGO for this test (default: bilinear) is the same as test_regrid_bilinear_2
```